scripts/test-helpers

```diff
@@ -480,6 +486,50 @@ export -f provision-cluster
 
 # ---
 
+provision-registry-proxy() {
+    set -e -o pipefail
```

Comment:
pipefail is probably a good thing to have in general, not sure if anything depends on that. if it is called that way in this function it has a global effect tho. is there anything particular in this function that requires it?

Reply:
I think I've refactored out anything that might have needed it in an earlier version. I can remove it if you think it'll cause problems?